### DISCUSSION TITLE
ModuleNotFoundError: No module named ‘pyam’ when importing Reporter

### DISCUSSION THREAD

**Discussion by Buaa-cy** [iiasa/message_ix#448] [Getting started] [answered]
While I running Tutorials, every time I import Reporter, Errors happened.
![屏幕截图 2021-03-06 162628](https://user-images.githubusercontent.com/61269536/110200397-bdd65080-7e98-11eb-9bd3-80ea8ce3bbe5.png)
Can you help me ,thanks!

**Comment by khaeru** [comment, marked as answer]
When you see `ModuleNotFoundError`, you should try to install the package that is mentioned. In this case, it's https://pyam-iamc.readthedocs.io/en/stable/install.html
